site/src/components/indicator/elderRay.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: component
+title: Elder Ray
+component: indicator/algorithm/elderRay.js
+tags:
+  - frontpage
+  - playground 
+namespace: indicator
+
+example-code: |
+  // Create and apply the elder ray algorithm
+  var elderRayAlgorithm = fc.indicator.algorithm.elderRay().period(6);
+  elderRayAlgorithm(data);
+
+  // the elder ray is rendered on its own scale
+  var yScale = d3.scale.linear()
+      .domain([-5, 5])
```

Comment:
Not always appropriate, but appropriate here. I didn't use extent as the range data is inside the 'elderRay' object on the data, and I didn't want to pollute the example. Do you think that should be included?

As for extent I'll raise an issue.